pkg/webhook/pod/mutating/sidecarset.go

```diff
@@ -437,10 +437,14 @@ func buildSidecars(isUpdated bool, pod *corev1.Pod, oldPod *corev1.Pod, matchedS
 				transferEnvs = util.MergeEnvVar(transferEnvs, injectedEnvs)
 				// insert volumes that initContainers used
 				for _, mount := range initContainer.VolumeMounts {
-					volumesInSidecars = append(volumesInSidecars, *volumesMap[mount.Name])
+					if vol, ok := volumesMap[mount.Name]; ok {
```

Comment:
maybe we can log a warning message and continue if mount.Name not exists in volumes. In practice, sidecarset webhook will validate that the volume of sidecar volumeMounts exists in sidecarVolumes. If it is not the case, some abnormal must occur e.g.  someone had remove webhookconfiguration of sidecarset

Reply:
> maybe we can log a warning message and continue if mount.Name not exists in volumes. In practice, sidecarset webhook will validate that the volume of sidecar volumeMounts exists in sidecarVolumes. If it is not the case, some abnormal must occur e.g. someone had remove webhookconfiguration of sidecarset

done